src/main/telemetry/ibus_shared.c

```diff
@@ -152,7 +152,11 @@ static uint8_t dispatchMeasurementRequest(ibusAddress_t address) {
     } else if (SENSOR_ADDRESS_TYPE_LOOKUP[address].value == IBUS_MEAS_VALUE_RPM) {
         return sendIbusMeasurement2(address, (uint16_t) (rcCommand[THROTTLE]));
     } else if (SENSOR_ADDRESS_TYPE_LOOKUP[address].value == IBUS_MEAS_VALUE_EXTERNAL_VOLTAGE) { //VBAT
-        return sendIbusMeasurement2(address, getBatteryVoltage());
+        if (telemetryConfig()->report_cell_voltage == 1) {
```

Comment:
Boolean `true` is not always equal to `1`. You should test for non-zero: ` if (telemetryConfig()->report_cell_voltage)`

Reply:
Done